source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/UnsupportedtypesParsingTest.cpp

```diff
@@ -176,7 +176,8 @@ namespace AdaptiveCardsSharedModelUnitTest
             Json::FastWriter writer;
             auto expectedAsString = writer.write(ParseUtil::GetJsonValueFromString(testJsonString));
             auto serializedCardAsString = writer.write(parseResult->GetAdaptiveCard()->SerializeToJsonValue());
-            Assert::AreEqual(expectedAsString, serializedCardAsString);
+            Assert::AreEqual(expectedAsString.substr(0, expectedAsString.size() - 1),
+                serializedCardAsString.substr(0, serializedCardAsString.size() - 1));
```

Comment:
what is this fixing?

Reply:
there is new line char at the end.

---
In reply to: [259075442](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r259075442) [](ancestors = 259075442)